docs/source/tutorials_source/platform/tutorial_active_learning.py

```diff
@@ -0,0 +1,191 @@
+"""
+
+Tutorial 2: Active Learning
+==============================================
+
+In this tutorial, we will run an active learning loop using both the lightly package and the platform.
+An active learning loop is a sequence of multiple samplings each choosing only a subset
+of all samples in the dataset. This workflow has the following structure:
+
+1. Choose an initial subset of your dataset e.g. using one of our samplers like the coreset sampler.
+    Split your dataset accordingly into a labeled set and unlabeled set.
+Next, a loop with following steps occurs:
+2. Train a classifier on the labeled set.
+3. Use the classifier to predict on the unlabeled set.
+4. Calculate active learning scores from the prediction.
+5. Use an active learning agent to choose the next samples to be labeled based on the active learning scores.
+6. Update the labeled set to include the newly chosen samples and remove them from the unlabeled set.
+
+In this tutorial we use a k-nearest-neighbor classifier that predicts the class of a sample
+based on the class of the k nearest samples in the labeled set.
+We use the euclidean distance between a sample's embeddings as the distance metric.
+The advantage of such a classifier compared to CNNs is that it is very fast and easily implemented.
+
+
+What you will learn
+-------------------
+* You learn how an active learning loop is set up and which components are needed for it.
+* You learn how to perform active learning with Lightly.
+
+Requirements
+------------
+- Make sure you are familiar with the command line tools described at https://docs.lightly.ai/getting_started/command_line_tool.html#
+- To make the definition of the dataset for training the classifier easy,
+we recommend using a dataset where the samples are grouped into folder according to their class.
+We use the clothing-dataset-small. You can download it using
+`git clone https://github.com/alexeygrigorev/clothing-dataset-small.git`
+
+"""
+
+
+# %%
+# Creation of the dataset on the lightly platform with embeddings
+# To perform samplings, we need to perform several steps, ideally with the CLI.
+# More documentation on each step is found at https://docs.lightly.ai/getting_started/command_line_tool.html#
+# A. Train a model on the dataset, e.g. with
+#    ```lightly-train input_dir="path/to/clothing-dataset-small trainer.max_epochs=5"```
+# B. Create embeddings for the dataset, e.g. with
+#    ```lightly-embed input_dir="path/to/clothing-dataset-small/train" checkpoint=mycheckpoint.ckpt```
```

Comment:
Further down we now also use an embedding for the test set. However, we never tell the user to create one. If we use a test set we should mention here that the user should also create embeddings for it.